src/ProductCard/index.tsx

```diff
@@ -55,22 +67,48 @@ const ProductCard: FunctionComponent<Props> = ({
 
   const ratings = rating && (
     <div className={styles['rating-row']}>
-      <Rating data-testid="star-rating" name="product-rating" value={rating} readOnly size="sm" />
+      <Rating
+        data-testid="star-rating"
+        name="product-rating"
+        value={rating}
+        className={styles['rating-row-stars']}
+        readOnly
+        size="sm"
+      />
       <span className={styles.reviews}>
         <small className={styles['average-rating']}>{rating}/5</small>{' '}
         <small data-testid="review-count">({reviews})</small>
       </span>
     </div>
   )
 
+  const deliveryDetails = (
+    <div className={styles['delivery-details']}>
+      {financeLink && (
+        <p className={styles['delivery-details-financing']}>
+          Special Financing for up to 12 months* <a href={financeLink}>Details</a>
+        </p>
+      )}
+      {deliveryDate && (
+        <p className={styles.delivery}>
+          Delivery as soon as: <span className={styles['delivery-date']}>{deliveryDate}</span>
+        </p>
+      )}
+    </div>
+  )
+
   const priceInfo = (
     <div className={styles['price-container']}>
+      <span className={styles.mobileBtnContainer}>
+        <StoreLocationButton storeLocation={storeLocation} onClick={storeLocationBtnOnClick} />
```

Comment:
You can make a prop for this to be passed down. It IS possible to pass down react components like Children. You could even do render props. Regardless, I wouldn't include this in the CL